problems/[230612] Implement Queue using Stacks.js

```diff
@@ -0,0 +1,60 @@
+const MyQueue = function () {
+  this.stack1 = [];
+  this.stack2 = [];
+};
+
+/**
+ * @param {number} x
+ * @return {void}
+ */
+MyQueue.prototype.push = function (x) {
+  this.stack1.push(x);
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.pop = function () {
+  if (!this.stack1.length && !this.stack2.length) {
+    return null;
+  }
+
+  if (!this.stack2.length) {
+    while (this.stack1.length) {
+      this.stack2.push(this.stack1.pop());
+    }
+  }
+
+  return this.stack2.pop();
+};
+
+/**
+ * @return {number}
+ */
+MyQueue.prototype.peek = function () {
+  if (this.stack2.length) {
+    return this.stack2[this.stack2.length - 1];
+  } else {
+    return this.stack1[0];
+  }
```

Comment:
```suggestion
  return this.stack2.length
    ? this.stack2[this.stack2.length - 1]
    : this.stack1[0];
```

삼항연산자로 간단하게 처리할 수 있을 것 같아요~